snap/snapcraft.yaml

```diff
@@ -0,0 +1,202 @@
+  gtk-3-themes:
+    interface: content
+    target: $SNAP/data-dir/themes
+    default-provider: gtk-common-themes:gtk-3-themes
+  icon-themes:
+    interface: content
+    target: $SNAP/data-dir/icons
+    default-provider: gtk-common-themes:icon-themes
+  sound-themes:
+    interface: content
+    target: $SNAP/data-dir/sounds
+    default-provider: gtk-common-themes:sounds-themes
+
+apps:
+    mscore:
+        adapter: full
+        command: usr/bin/mscore
+        command-chain:
+            - bin/desktop-launch
+        environment:
+            # Make this the default till Qt apps in wayland look & work better
+            DISABLE_WAYLAND: 1
+            QT_QPA_PLATFORMTHEME: gtk3
+        plugs:
+          - gsettings
+          - home
+          - network
+          - pulseaudio
+          - audio-playback
+          - unity7
+          - x11
+          - opengl
+          - cups-control
+          - alsa
+          - desktop
+          - wayland
+          - desktop-legacy
+          - removable-media
+
+plugs:
+  # gtk-common-themes support
+  gtk-3-themes:
+    interface: content
+    target: $SNAP/data-dir/themes
+    default-provider: gtk-common-themes:gtk-3-themes
+  icon-themes:
+    interface: content
+    target: $SNAP/data-dir/icons
+    default-provider: gtk-common-themes:icon-themes
+  sound-themes:
+    interface: content
+    target: $SNAP/data-dir/sounds
+    default-provider: gtk-common-themes:sounds-themes
+
+parts:
+    musescore:
+        source: https://github.com/musescore/MuseScore
+        source-type: git
+        plugin: make
+        override-build: |
+            make revision
+            QT_SELECT=qt5 snapcraftctl build
+        override-pull: |
+            snapcraftctl pull
+            "$SNAPCRAFT_STAGE"/scriptlets/selective-checkout \
+                        --release-tag-prefix='v' \
+                        --stable-tag-pattern=[[:digit:]]+$ \
+                        --force-stable
+        build-packages:
+            #- extra-cmake-modules
+            #- doxygen
+            # From https://musescore.org/en/handbook/developers-handbook/compilation/compile-instructions-ubuntu-1604lts-and-1804lts-git
+            # Tools
+            - cmake
+            - g++
+            # Audio & others
+            - libasound2-dev
+            - portaudio19-dev
+            - libmp3lame-dev
+            - libsndfile1-dev
+            - libportmidi-dev
+            - libssl-dev
+            - libpulse-dev
+            - libfreetype6-dev
+            - libfreetype6
+            # «Failed to find "GL" in "".»
+            - libdrm-dev
+            - libgl1-mesa-dev
+            - libegl1-mesa-dev
+            # QT
+            - qtbase5-dev
+            - qttools5-dev
+            - qttools5-dev-tools
+            - qtwebengine5-dev
+            - qtscript5-dev
+            - libqt5xmlpatterns5-dev
+            - libqt5svg5-dev
+            - libqt5webkit5-dev
+            - qtquickcontrols2-5-dev
```

Comment:
I believe "qml-module-qtquick-controls" is neede too, see e.g. https://musescore.org/en/node/301501#comment-980984